nvme-print-stdout.c

```diff
@@ -2724,13 +2724,23 @@ static void stdout_id_ns_descs(void *data, unsigned int nsid)
 	printf("NVME Namespace Identification Descriptors NS %d:\n", nsid);
 	for (pos = 0; pos < NVME_IDENTIFY_DATA_SIZE; pos += len) {
 		struct nvme_ns_id_desc *cur = data + pos;
+		const char *nidt_type = NULL;
 
 		if (cur->nidl == 0)
 			break;
 
+		if (verbose) {
+			printf("loc     : %d\n", pos);
+			printf("nidt    : %d\n", (int)cur->nidt);
+			printf("nidl    : %d\n", (int)cur->nidl);
+		}
+
 		switch (cur->nidt) {
 		case NVME_NIDT_EUI64:
 			memcpy(eui64, data + pos + sizeof(*cur), sizeof(eui64));
+			nidt_type = "eui64";
+			if (verbose)
+				printf("type    : %s\n", nidt_type);
```

Comment:
Why not just `printf("type : eui64\n")` (or `puts()`)? That would allow you to get rid of the `nidt_type` variable.

Reply:
Also seems like this is redundant with the `printf("eui64 : ");` printed immediately afterwards?